docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -389,19 +436,31 @@ export default function AppLayoutDocsFooter() {
                 margin="dense"
                 name="comment"
                 fullWidth
-                rows={6}
+                rows={4}
                 value={comment}
                 onChange={handleChangeTextfield}
                 inputProps={{
-                  'aria-label': t('feedbackCommentLabel'),
                   'aria-describedby': 'feedback-description',
                   ref: inputRef,
                 }}
               />
+              {rating !== 1 && (
+                <Typography id="feedback-description" color="text.secondary">
+                  {t('feedbackMessageToGithub.main')}{' '}
+                  <Link
+                    href={`${process.env.SOURCE_CODE_REPO}/issues/new?template=4.docs-feedback.yml&page-url=${window.location.href}`}
```

Comment:
The handling of the hash is not strong here, feedback hash != URL hash. We could add the `feedbackHash` on the page URL parameter and remove the hash from the current URL.

Reply:
I think that processing the url could lead to loosing precious information in some edge cases. For example when their is a query: https://mui.com/material-ui/material-icons/?query=snow

I prefer to fill the issue with raw data, assuming that if it's a bug, user will describe which demo is buggy, or which sentence is incorrect.

Reply:
+1 to keep the query parameters. My issue is with the hash. e.g. https://mui-org.slack.com/archives/C041SDSF32L/p1668885004082799, I understand the feedback as unrelated to #testing-caveats. But now that developers can leave feedback on specific hash. How do we get this information in Slack?

Reply:
- On slack: It's added at the end like in [this test](https://mui-org.slack.com/archives/C041SDSF32L/p1666696140121439)
> sent from [the href with hash] (section [section name]) 

- On github: It's not sent, because if the user takes time to fill an issue, they will be precise enough